IntegrationTests/Tests/AccountTests.cs

```diff
@@ -13,54 +14,54 @@ public class AccountTests(VitalApiFactory vaf) : TestBase(vaf)
     [Fact]
     public async Task Forgot_Password_input_not_address_return_400()
     {
-        var forgotPassordDto = new ForgotPasswordDto()
+        var forgotPasswordDto = new ForgotPasswordDto()
         {
             Email = "userapp"
         };
 
         var response =
-            await _client.PostAsync("/Identity/Account/Forgot-Password", JsonContent.Create(forgotPassordDto));
+            await _client.PostAsync("/Identity/Account/Forgot-Password", JsonContent.Create(forgotPasswordDto));
 
         response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
     }
 
     [Fact]
     public async Task Forgot_Password_user_not_found_return_200()
     {
-        var forgotPassordDto = new ForgotPasswordDto()
+        var forgotPasswordDto = new ForgotPasswordDto()
         {
             Email = "forgot-password-not-found@app.com"
         };
 
-        _dbContext.Users.FirstOrDefault(u => u.UserName == forgotPassordDto.Email).Should().BeNull();
+        _dbContext.Users.FirstOrDefault(u => u.UserName == forgotPasswordDto.Email).Should().BeNull();
 
         var response =
-            await _client.PostAsync("/Identity/Account/Forgot-Password", JsonContent.Create(forgotPassordDto));
+            await _client.PostAsync("/Identity/Account/Forgot-Password", JsonContent.Create(forgotPasswordDto));
 
         response.StatusCode.Should().Be(HttpStatusCode.OK);
     }
 
     [Fact]
     public async Task Forgot_Password_return_500()
```

Comment:
why is this called return_500 if the assertion is for OK?

Reply:
ask mr Mazur, i will change it though